src/cltkv1/morphology/Morphosyntax.py

```diff
@@ -0,0 +1,330 @@
+    exclusive = auto()
+    inclusive = auto()
+    
+verbal_features = [VerbForm, Tense, Mood, Aspect, Voice, Person, Polarity, Politeness, Clusivity, Evidentiality]
+    
+# Nominal features
+    
+class Case(MorphosyntacticFeature):
+    # structural cases
+    nominative = auto()
+    accusative = auto()
+    ergative = auto()
+    absolutive = auto()
+    
+    # oblique cases
+    abessive = auto()
+    befefactive = auto()
+    causative = auto()
+    comparative = auto()
+    considerative = auto()
+    comitative = auto()
+    dative = auto()
+    distributive = auto()
+    equative = auto()
+    genitive = auto()
+    instrumental = auto()
+    partitive = auto()
+    vocative = auto()    
+    
+    # spatiotemporal cases
+    ablative = auto()
+    additive = auto()
+    adessive = auto()
+    allative = auto()
+    delative = auto()
+    elative = auto()
+    essive = auto()
+    illative = auto()
+    inessive = auto()
+    lative = auto()
+    locative = auto()
+    perlative = auto()
+    sublative = auto()
+    superessive = auto()
+    terminative = auto()
+    temporal = auto()
+    translative = auto()
+    
+class Gender(MorphosyntacticFeature):
+    masculine = auto()
+    feminine = auto()
+    neuter = auto()
+    common = auto()
+    
+class Animacy(MorphosyntacticFeature):
+    animate = auto()
+    human = auto()
+    inaninate = auto()
+    non_human = auto()
+    
+class Number(MorphosyntacticFeature):
+    collective = auto()
+    count_plural = auto()
+    dual = auto()
+    greater_paucal = auto()
+    greater_plural = auto()
+    inverse_number = auto()
+    paucal = auto()
+    plural = auto()
+    plurale_tantum = auto()
+    singular = auto()
+    trial = auto()
+    
+class Definiteness(MorphosyntacticFeature):
+    complex = auto()
+    construct_state = auto()
+    definite = auto()
+    indefinite = auto()
+    specific_indefinite = auto()
+    
+class Degree(MorphosyntacticFeature):
+    absolute_superlative = auto()
+    comparative = auto()
+    equative = auto()
+    positive = auto()
+    superlative = auto()
+    
+nominal_features = [Case, Gender, Animacy, Number, Definiteness, Degree]
+    
+    
+# Other lexical features
+    
+class PrononimalType(MorphosyntacticFeature):
+    article = auto()
+    demonstrative = auto()
+    emphatic = auto()
+    exclamative = auto()
+    indefinite = auto()
+    interrogative = auto()
+    negative = auto()
+    personal = auto()
+    reciprocal = auto()
+    relative = auto()
+    total = auto() 
+    
+class Possessive(PrivativeFeature):
+    pass
+    
+class Numeral(MorphosyntacticFeature):
+    cardinal = auto()
+    distributive = auto()
+    fractional = auto()
+    multiplicative = auto()
+    ordinal = auto()
+    range = auto()
+    sets = auto()
+    
+class Reflexive(PrivativeFeature):
+    pass
+
+class Foreign(PrivativeFeature):
+    pass
+
+class Abbr(PrivativeFeature):
+    pass
+
+class Typo(PrivativeFeature):
+    pass

```

Comment:
I have a general comment: I see what these terms mean, but there should be a link directly in code to show where they come from and why such choice was made. You mentioned this https://universaldependencies.org/u/feat/index.html. Maybe, next to "participle", you can add this link https://universaldependencies.org/u/feat/VerbForm.html#Part.

Is it a good idea to put such links inside the code? Good because we can directly get why it's here, but the link might change or redirect to nowhere and it would break the comment.

Reply:
Stout point.  Will add links and some descriptions.